### DISCUSSION TITLE
How to debug a CollaborationPlugin issue?

### DISCUSSION THREAD

**Discussion by 6peterlu** [facebook/lexical#3641] [Q&A]
I have a LexicalComposer that has a CollaborationPlugin in it. I can see edits from both editors inside the Websockets by using the Chrome network inspector. However, the edits are not syncing in the LexicalComposer.

How do I go about debugging this? Further confusing me is that I have another LexicalComposer element that has no issues with collaborative syncing.

**Comment by 6peterlu** [comment]
My guess is that somehow `syncYjsChangesToLexical` inside `SyncEditorStates.ts` is not working as expected, but I'm not certain about that.

**Comment by fantactuka** [comment]
You could start with checking whether data flows into the plugin (adding breakpoints into syncYjsChangesToLexical and syncLexicalUpdateToYjs). If you're not receiving any updates in there, ensure you pass document (room) id to the plugin and that yDoc is created in yDocMap

Also, any chance you can have sandbox with code you're using?
